docs/framework/whats-new/obsolete-members.md

```diff
@@ -329,7 +329,7 @@ The tables in this article list the type members that are obsolete in the .NET F
   
 |Type|Member|Message|  
 |----------|------------|-------------|  
-|<xref:System.Diagnostics.Eventing.Reader.StandardEventKeywords?displayProperty=nameWithType>|<xref:System.Diagnostics.Eventing.Reader.StandardEventKeywords>|First deprecated in the .NET Framework 4.5.<br /><br /> Incorrect value; use <xref:System.Diagnostics.Eventing.Reader.StandardEventKeywords> instead.|  
+|<xref:System.Diagnostics.Eventing.Reader.StandardEventKeywords?displayProperty=nameWithType>|<xref:System.Diagnostics.Eventing.Reader.StandardEventKeywords.CorrelationHint>|First deprecated in the .NET Framework 4.5.<br /><br /> Incorrect value; use <xref:System.Diagnostics.Eventing.Reader.StandardEventKeywords> instead.|  
```

Comment:
last xref should be System.Diagnostics.Eventing.Reader.StandardEventKeywords.CorrelationHint2